migrations/sql/006-rename-eventtype.down.sql

```diff
@@ -0,0 +1,2 @@
+UPDATE events SET type=feature-revive WHERE type=feature-revived;
+UPDATE events SET type=feature-archive WHERE type=feature-archived;
```

Comment:
The types need to be single-quoted. Have you run the migration locally (`db-migrate up`)?